src/System.Console/src/System.Console.csproj

```diff
@@ -53,22 +53,19 @@
   </ItemGroup>
   <ItemGroup Condition=" '$(OS)' == 'Unix' ">
     <Compile Include="System\ConsolePal.Unix.cs" />
-    <Compile Include="Interop\Interop.manual.Unix.cs" />
-    <Compile Include="$(CommonPath)\System\IO\StringBuilderCache.cs">
-      <Link>Common\System\IO\StringBuilderCache.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)\Interop\Interop.CoreFileIO.Unix.cs">
-      <Link>Common\Interop\Interop.CoreFileIO.Unix.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)\Interop\Interop.Errors.Unix.cs">
-      <Link>Common\Interop\Interop.Errors.Unix.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)\Interop\Interop.strerror.Unix.cs">
-      <Link>Common\Interop\Interop.strerror.Unix.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)\Microsoft\Win32\SafeHandles\SafeFileHandle.Unix.cs">
-      <Link>Common\Microsoft\Win32\SafeHandles\SafeFileHandle.Unix.cs</Link>
-    </Compile>
+    <Compile Include="$(CommonPath)\Microsoft\Win32\SafeHandles\SafeFileHandle.Unix.cs"><Link>Common\Microsoft\Win32\SafeHandles\SafeFileHandle.Unix.cs</Link></Compile>
+    <Compile Include="$(CommonPath)\System\IO\StringBuilderCache.cs"><Link>Common\System\IO\StringBuilderCache.cs</Link></Compile>
+    
```

Comment:
I'm not sure I think the Link on the same line is better. Does VS reformat these when you save the project there? if so I would just put them on their own lines.